packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -82,26 +83,25 @@ export class RemoteTableService {
       workspaceId,
     );
 
-    if (currentRemoteTables.length === 0 || !shouldFetchPendingSchemaUpdates) {
-      const distantTablesWithStatus = Object.keys(distantTables).map(
-        (tableName) => ({
+    const shouldGetTablesWithPendingSchemaUpdates =
```

Comment:
I have trouble understanding this function overall, why ByServerId in the naming?

also, what does shouldFetchPendingSchemaUpdates means, it's not clear to me

Reply:
oh is actually shouldGetPendingSchemaUpdates?

Reply:
serverId can definitely be removed. Will discuss it with @ijreilly because the name is not the same in the resolver already

shouldFetchPendingSchemaUpdates is a boolean in input that says if you should return the pending schema updates in the response or not

Reply:
Updated!